src/META-INF/plugin.xml

```diff
@@ -61,6 +67,14 @@
     <internalFileTemplate name="Go File"/>
 
     <!-- project -->
+    <toolWindow id="Go" anchor="bottom" icon="/icons/go.png" secondary="true" factoryClass="com.goide.ui.GoToolWindowFactory" />
+
+    <projectConfigurable instance="com.goide.ui.GoConfigurable" groupId="tools"/>
+
+    <projectService
```

Comment:
@ignatov no

@dlsniper put it to "language" group, it's sad to have go-related configurables in different places, I'll made a Go-group later. Also meaningful name for configurable is welcome.